README.md

```diff
@@ -56,6 +56,29 @@ composer require --dev friendsofphp/php-cs-fixer
 For more details and other installation methods, see
 [installation instructions](./doc/installation.rst).
 
+### Run with Docker
```

Comment:
```suggestion
### Run with Docker (early access)
```

only idea. I'm not sure if we confident all will work great with very first version of it, or maybe we mention to users it's early access here and they could experience some imperfections

Reply:
I am pretty sure it will work as expected, because images are currently built [in the @hgiesenow's fork](https://github.com/elbformat/PHP-CS-Fixer/actions/workflows/docker.yml) and you can pull them from the registry and verify 🙂.

Reply:
I would also recommend not to add this. Otherwise we would need to define, when the early access ends and why. I think we should instead iteratively improve it, as soon as there are issues appearing.